text/0112-server-runtime.md

```diff
@@ -0,0 +1,329 @@
+---
+title: LWC server runtime
+status: DRAFTED
+created_at: 2020-01-06
+updated_at: 2020-01-06
+pr: https://github.com/salesforce/lwc-rfcs/pull/23
+---
+
+# LWC server runtime
```

Comment:
@pmdartus "LWC Server Runtime" sounds like an application framework for running LWC, like LWR (and runtime is super overloaded). Can we rename this to "LWC Server Engine" or "LWC DOM Decoupling"